client/src/components/PizzaPlaceShow.js

```diff
@@ -1,7 +1,11 @@
 import React, { useEffect, useState } from "react"
 import ReviewItem from "./ReviewItem.js"
+import NewReviewForm from "./NewReviewForm.js"
 
 const PizzaPlaceShow = props => {
+
+	const pizzaId = props.match.params.id
```

Comment:
unless this `pizzaId` is used elsewhere in this file, we can keep it defined in the function that actually needs it - the getPizzaPlace